.github/workflows/docker-release.yml

```diff
@@ -29,22 +29,38 @@ env:
   WITH_PUSH: ${{ github.event_name == 'push' && (github.event.ref == 'refs/heads/nightly' || startsWith(github.event.ref, 'refs/tags/v')) }}
 
 jobs:
+  generate-matrix:
+    if: github.repository_owner == 'pytorch'
+    runs-on: [self-hosted, linux.large]
+    outputs:
+      release-matrix: ${{ steps.generate-matrix.outputs.release-matrix }}
+    steps:
+      - name: Checkout PyTorch
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
+        with:
+          fetch-depth: 1
+          submodules: false
+      - name: Get docker release matrix
+        id: generate_matrix
```

Comment:
Look like there is naming mismatches here?

* `matrix` v.s. `release-matrix`
* `generate_matrix` v.s. `generate-matrix`